batch/batch/driver/instance_collection/job_private.py

```diff
@@ -180,7 +179,8 @@ async def schedule_with_error_handling(app, record, id, instance):
 
         await waitable_pool.wait()
 
-        log.info(f'attempted to schedule {n_records_seen} jobs for {self}')
+        if n_records_seen > 0:
+            log.info(f'attempted to schedule {n_records_seen} jobs for {self}')
```

Comment:
As a general comment, I don't think gating log statements on these conditions will help much. I suspect most of our log volume comes when the system is under heavy load, i.e. n_records_seen >>> 0.